README.md

```diff
@@ -513,6 +513,131 @@ The option **`T`** specifies the temperature of the simulation, and the option *
 
 The **`atom_type`** set the type for the atoms in the system. The names of the atoms are those provided in the `conf_file` file. The **`atom_mass`** set the mass for the atoms. Again, the name of the atoms are those provided in the `conf_file`.
 
+# Multi-GPU support
+Recently, we updated code for multi-GPU support. The main changes are focused on the interface part of LAMMPS and TensorFlow,
+while the LAMMPS package was also minorly modified. Here, we'll give you a brief overview of this upgrade and provide a simple test. First, you need to install the GPU support environment for DeePMD-kit.
+## Install GPU support environments for DeePMD-kit
+First, you need a CUDA environment, and CUDA-10.0 is required. If you have a higher version of CUDA, such as CUDA-10.1, you can use it when compiling or running the DeePMD-kit's c-plus-plus interface. However, when you use the DeePMD-kit's python interface, the CUDA-10.0 environment is required. Lower versions of CUDAs are not recommended for use.
+
+For a successful installation, we strongly recommend that you use Bazel-0.24.1, TensorFlow-1.14.0-GPU, as well as higher versions of CMake and git. Sometimes you may also report an error due to a low python version or a GCC version issue. When you have a compilation problem, it may be helpful to try to upgrade the software version.
+
+Detailed installation process can be referred to [tf-1.14-gpu](doc/install-tf.1.14-gpu.md).
+## Code upgrade
+We'll briefly describe this upgrade in three parts.
+### Allocate GPU resources to TensorFlow
+Tensorflow uses all available GPU resources by default. So in the original code, when we run parallel programs, multiple processors apply for memory resources will conflict and report errors. So we use the TensorFlow graph API to assign a specific GPU to each TensorFlow graph based on the device ranks while limiting the default memory usage of TensorFlow in multiple GPU cases. The code can be viewed [here](source/lib/src/NNPInter.cc), focusing mainly on the init function.
+### Get the processor's node rank
+When working on platforms across nodes, we need to consider how to get the device rank mentioned in the previous section. If you're using Open MPI, it comes with a node-rank API, but if you use an Intel impi, you may need to use another method to specify node-rank. At present, we think it is a good way to be compatible with multi-platforms by dividing the MPI communicator based on the processor name. The code can be viewed [here](source/lmp/pair_nnp.cpp), focusing mainly on the get_node_rank function.
+### Cmake conditional compilation
+We introduced the USE-CUDA-TOOLKIT parameter as a control variable for whether to compile using the CUDA environment. If you want to build DeePMD-kit with CUDA-toolkit support, then execute cmake
+```bash
+cmake -DTF_GOOGLE_BIN=true -DUSE_CUDA_TOOLKIT=true -DTENSORFLOW_ROOT=$tensorflow_root \
```

Comment:
So, we don’t need to specify the CUDA’s PATH? Does it just use TensorFlow’s settings?

Reply:
Cmake has a function, named "find_package(CUDA REQUIRED)", which can detect the CUDA environment by default. If you load the CUDA module or set the CUDA environment in your system, when you set "-DUSE_CUDA_TOOLKIT=true" , CMake will set the CUDA environment by default.

Reply:
user can specify the cuda tool-kit path by CUDA_TOOLKIT_ROOT_DIR, please check
https://cmake.org/cmake/help/latest/module/FindCUDA.html